support-frontend/assets/components/orderSummary/contributionsOrderSummary.tsx

```diff
@@ -245,25 +256,17 @@ export function ContributionsOrderSummary({
 			<div css={[summaryRow, rowSpacing, boldText, totalRow(!!tsAndCs)]}>
 				<p>Total</p>
 				<p>
-					{formattedPromotionAmount && (
+					{discountPrice && (
```

Comment:
There are two different conditional blocks here. One where `discountPrice` is truthy and one where it isn't (`discountPrice &&` and `!discountPrice &&`). So it's basically an if else. I wonder if using a ternary would make things clearer? Or even extract out to a smaller local component (`PriceSummary`?) which could use an actual if/else statement, returning different things in each branch.

Reply:
Have created `priceSummary` component with an if/else statement now, hopefully its what your expecting. Looks more readable now would say. Updated.